File: atcoder/abc370a_raise_both_hands.hs
-- vicfred
-- https://atcoder.jp/contests/abc370/tasks/abc370_a
-- implementation
main :: IO ()
main = getLine >>= putStrLn . f . map read . words

f :: [Int] -> String
f [1, 0] = "Yes"
f [0, 1] = "No"
f _      = "Invalid"

File: atcoder/abc370a_raise_both_hands.cpp
// vicfred
// https://atcoder.jp/contests/abc370/tasks/abc370_a
// implementation
#include <iostream>

using namespace std;

int main() {
  int L, R;
  cin >> L >> R;
  if((L ^ R) == 1) {
    if(L == 1) {
      cout << "Yes" << endl;
    } else {
      cout << "No" << endl;
    }
  } else {
    cout << "Invalid" << endl;
  }
  return 0;
}
